packages/tensorboard/tensorboard.0.0.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+bug-reports:  "https://github.com/LaurentMazare/ocaml-tensorboard/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-tensorboard"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-tensorboard.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "-p" name "-j" jobs]]
+depends: [
+  "base" {>= "0.11.0"}
```

Comment:
This was wrong. It should be `v0.11.0` not `0.11.0`. I'll fix that here directly. Could you fix it upstream too?

Reply:
Thanks!